blazor/diagram/tool-tip.md

```diff
@@ -19,4 +19,656 @@ By default, the diagram displays a tooltip that provides information about the s
+    private void PositionChange()
+    {
+        connector[0].Tooltip.Position = Position.RightCenter;
+    }
+}
+```
+
+## How to set tooltip content for nodes/connectors
+
+A text or a piece of information assigned to the Tooltip’s [Content](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.DiagramTooltip.html#Syncfusion_Blazor_Diagram_DiagramTooltip_Content) property will be displayed as the main content of the Tooltip for nodes/connectors.
+
+The following code example is used to set tooltip content for nodes.
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+@using Syncfusion.Blazor.Popups
+<input type="button" value="Node Content" @onclick="ContentChange" />
+<SfDiagramComponent Width="1000px" Height="500px" Nodes="@nodes" />
+@code
+{
+    DiagramObjectCollection<Node> nodes;
+
+    protected override void OnInitialized()
+    {
+        nodes = new DiagramObjectCollection<Node>();
+        Node node = new Node()
+        {
+            ID = "node1",
+            OffsetX = 250,
+            OffsetY = 250,
+            Width = 100,
+            Height = 100,
+            Style = new ShapeStyle() 
+            { 
+                Fill = "#6495ED", 
+                StrokeColor = "white" 
+            },
+            Tooltip = new DiagramTooltip(){Content="NodeTooltip"},
+            Constraints = NodeConstraints.Default|NodeConstraints.Tooltip,
+        };
+        nodes.Add(node);
+    }
+    //Change Content at run time
+    private void ContentChange()
+    {
+        nodes[0].Tooltip.Content = "UpdateTooltipContent";
+    }
+}
+```
+
+The following code example is used to set tooltip content for connectors.
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+@using Syncfusion.Blazor.Popups
+<input type="button" value="Connector Content" @onclick="ContentChange" />
+<SfDiagramComponent Width="1000px" Height="500px" Connectors="connector" />
+@code
+{
+    //Defines diagram's connector collection.
+    DiagramObjectCollection<Connector> connector;
+
+    protected override void OnInitialized()
+    {
+        connector = new DiagramObjectCollection<Connector>();
+        Connector connectors = new Connector()
+        {
+            ID = "Connector1",
+            SourcePoint = new DiagramPoint() { X = 500, Y = 500 },
+            TargetPoint = new DiagramPoint() { X = 600, Y = 400 },
+            Tooltip = new DiagramTooltip(){Content="ConnectorTooltip"},
+            Constraints =  ConnectorConstraints.Default|ConnectorConstraints.Tooltip ,
+        };
+        connector.Add(connectors);
+    }
+    //Change Content at run time
+    private void ContentChange()
+    {
+        connector[0].Tooltip.Content = "UpdateTooltipContent";
+    }
+}
+```
+
+## How to shows/hides the tip pointer for tooltip
+
+[ShowTipPointer](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.DiagramTooltip.html#Syncfusion_Blazor_Diagram_DiagramTooltip_ShowTipPointer) property value is true, if the tip pointer is visible; otherwise, false.
```

Comment:
The [ShowTipPointer](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.DiagramTooltip.html#Syncfusion_Blazor_Diagram_DiagramTooltip_ShowTipPointer) property value is true if the tip pointer is visible; otherwise, false.